### PR TITLE
Improve MoveLiquidity event params

### PR DESCRIPTION
**Pull request by EdNoepel**
# Description of change
## High level
Update `PositionManager.MoveLiquidity` event to expose amount of LP redeemed from the _from_ bucket and awarded to the _to_ bucket.

# Description of bug or vulnerability and solution
Our indexer's `Lend` entity records an actor's LP whether or not they've memorialized their position into a position NFT. This breaks when moving liquidity in `PositionManager`, because the event does not reveal the amount of LP redeemed from the _from_ bucket or awarded to the _to_ bucket. We cannot query amounts from `Pool.lenderInfo` because the LP belongs to position manager.  We should not get this from the `Pool.MoveQuoteToken` event because we cannot reliably associate the _Pool_ event with the _PositionManager_ event.

# Contract size
## Pre Change
```
============ Deployment Bytecode Sizes ============
  PositionManager          -  18,787B  (76.44%)
```
## Post Change
```
============ Deployment Bytecode Sizes ============
  PositionManager          -  18,858B  (76.73%)
```

# Gas usage
## Pre Change
| src/PositionManager.sol:PositionManager contract |                 |        |        |         |         |
|--------------------------------------------------|-----------------|--------|--------|---------|---------|
| Function Name                                    | min             | avg    | median | max     | # calls |
| moveLiquidity                                    | 5864            | 235506 | 238824 | 621753  | 19      |

| src/RewardsManager.sol:RewardsManager contract |                 |         |         |         |         |
|------------------------------------------------|-----------------|---------|---------|---------|---------|
| Function Name                                  | min             | avg     | median  | max     | # calls |
| moveStakedLiquidity                            | 1819004         | 1971829 | 1971829 | 2124655 | 2       |

## Post Change
| src/PositionManager.sol:PositionManager contract |                 |        |        |         |         |
|--------------------------------------------------|-----------------|--------|--------|---------|---------|
| Function Name                                    | min             | avg    | median | max     | # calls |
| moveLiquidity                                    | 5864            | 235879 | 239281 | 622325  | 19      |

| src/RewardsManager.sol:RewardsManager contract |                 |         |         |         |         |
|------------------------------------------------|-----------------|---------|---------|---------|---------|
| Function Name                                  | min             | avg     | median  | max     | # calls |
| moveStakedLiquidity                            | 1821292         | 1974117 | 1974117 | 2126943 | 2       |

### CODE REVIEW

File: src/interfaces/position/IPositionManagerEvents.sol [hunk 1]
```diff
@@ -42,16 +42,20 @@ interface IPositionManagerEvents {
...
+     *  @param  lpAwardedTo    Amount of LP credited to the `to` bucket.
      */
     event MoveLiquidity(
         address indexed lender,
         uint256 tokenId,
         uint256 fromIndex,
-        uint256 toIndex
+        uint256 toIndex,
+        uint256 lpRedeemedFrom,
+        uint256 lpAwardedTo
```
Comment (MikeHathaway) [inline src/interfaces/position/IPositionManagerEvents.sol:58]: Do `lpRedeemedFrom` and `lpAwardedTo` ever differ? Could we use only one param instead?

Comment (EdNoepel) [reply src/interfaces/position/IPositionManagerEvents.sol:58]: Yes, LP will differ moving between buckets with different exchange rates.

**Review by grandizzy — APPROVED** [review]
lgtm

**Review by MikeHathaway — APPROVED** [review]
LGTM

**Pull request merged**